lib/wibox/init.lua

```diff
@@ -324,6 +329,8 @@ local function new(args)
     return ret
 end
 
+capi.drawin.add_signal("property::wibox")
+
```

Comment:
Did you intend to have this and the previous change in this commit? If so, why and what does it do?